inf-clojure.el

```diff
@@ -285,10 +285,11 @@ buffers after they are created with `rename-buffer'."
            (inf-clojure-repl-p))
       (setq inf-clojure-buffer (current-buffer))
     (let ((repl-buffers (inf-clojure-repls)))
-     (if (> (length repl-buffers) 0)
-         (when-let ((repl-buffer (completing-read "Select default REPL: " repl-buffers nil t)))
-           (setq inf-clojure-buffer (get-buffer repl-buffer)))
-       (user-error "No buffers have an inf-clojure process")))))
+      (if (> (length repl-buffers) 0)
+          (when-let ((repl-buffer (completing-read "Select default REPL: " repl-buffers nil t)))
+            (setq inf-clojure-buffer (get-buffer repl-buffer)))
+        (user-error "No buffers have an inf-clojure process"))))
+  (message "inf-clojure repl set to %s" inf-clojure-buffer))
```

Comment:
Probably it'd be better to use something like "Current/active inf-clojure REPL set to ...".

Reply:
Thanks for the input, I made the change!